src/test/resources/emails/sessionPublishedEmailForInstructor.html

```diff
@@ -1,14 +1,14 @@
 Hello Instructor1 Course1,
 
 <p/>
-The email below has been sent to students of course: idOfTypicalCourse1.<p/><br>
+The email below has been sent to students of course: idOfTypicalCourse1.<p/><br><br>=== Email message as seen by the students ===<br>
 
 <p/>
 The feedback responses for the following feedback session is now open for viewing.
 <br>&nbsp;&nbsp; Course: [idOfTypicalCourse1]Typical Course 1 with 2 Evals 
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>
-<strong>To view the responses for this session, please go to this Web address: </strong><a href="{The student's unique results url appears here}">{The student's unique results url appears here}</a>
+<strong>To view the responses for this session, please go to this Web address: </strong><a href="${app.url}/page/instructorFeedbackResultsPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session">{in the actual email sent to the students, this will be the unique link}</a>
```

Comment:
@tarun14110, Just to check, on master now, the link is not clickable. Is it deliberate that the link leads to the instructor's results/submit page?